pkg/apis/core/v1/types.go

```diff
@@ -405,6 +406,9 @@ type (
 
 		// This is the timeout setting for executor to wait for pod specialization.
 		SpecializationTimeout int
+
+		// Only for newdeploy to provide custom metrics support added in autoscaling/v2beta2
```

Comment:
nit: Can you please change the comment to start with `CustomMetrics` so that golint doesn't show any warnings?